opus/import/obs_cassini_common_pds4.py

```
@@ -8,10 +8,114 @@
+
+    def field_obs_instrument_coiss_shutter_state_id(self):
+        return self._create_mult(self._index_col('cassini:shutter_state_id'))
+
+    def field_obs_instrument_coiss_image_number(self):
+        return self._index_col('cassini:image_number')
+
+    def field_obs_instrument_coiss_instrument_mode_id(self):
+        return self._create_mult(self._index_col('cassini:instrument_mode_id'))
+
+    def field_obs_instrument_coiss_target_desc(self):
+        target_desc = self._index_col('cassini:pds3_​target_​desc')
```

Ruff (0.15.9)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)